python/revng/internal/cli/_commands/graphql/runner.py

```diff
@@ -150,7 +150,9 @@ async def runner(client: AsyncClientSession):
                 }
             }"""
             )
-            result = await client.execute(q, {**arguments, "target": targets, "index": index})
+            result = await client.execute(
```

Comment:
When running `revng check-conventions --force-format` on my machine this change gets reverted to the previous version, can you double-check that it doesn't on yours?